### PR TITLE
OHM-297 visualizer user pref

### PR DESCRIPTION
**Pull request by rcrichton**
This PR build off of https://github.com/jembi/openhim-console/pull/407

### CODE REVIEW

File: app/scripts/controllers/visualizer.js [hunk 1]
```diff
@@ -217,19 +217,56 @@ angular.module('openhimConsoleApp')
...
+      $scope.selectVis = function (vis, callback) {
+        if (!callback) {
+          callback = function () {};
+        }
+
+        // set current visualizer
+        $scope.selectedVis = vis;
+
+        // store in current session
+        if (! consoleSession.sessionUserSettings) {
```
Comment (BMartinos) [inline app/scripts/controllers/visualizer.js]: Not sure if there should be a space between ! and variable?

File: app/scripts/controllers/visualizer.js [hunk 2]
```diff
@@ -217,19 +217,56 @@ angular.module('openhimConsoleApp')
...
+          consoleSession.sessionUserSettings = {};
+        }
+        consoleSession.sessionUserSettings.selectedVisualizer = vis.name;
+        localStorage.setItem('consoleSession', JSON.stringify(consoleSession));
+        // store in user settings
+        if (! user.settings) {
```
Comment (BMartinos) [inline app/scripts/controllers/visualizer.js]: Same as above question

**Comment by BMartinos** [comment]
Hey @rcrichton This works really well for me!. Tested on various browsers and deleting sessions to see if settings gets pulled through and works as expected :+1:

File: app/scripts/controllers/visualizer.js (hunk 1 above)
Comment (rcrichton) [inline app/scripts/controllers/visualizer.js]: Agreed, that looks weird.

**Comment by rcrichton** [comment]
@BMartinos make the changes.

**Comment by BMartinos** [comment]
LGTM :+1:

**Pull request merged**